spec/helpers/activity_helper_spec.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the ActivityHelper. For example:
+#
+# describe ActivityHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe ActivityHelper, type: :helper do
+end
```

Comment:
Coveralls wants test coverage for the ActivityHelper:
https://coveralls.io/builds/36275346/source?filename=app%2Fhelpers%2Factivity_helper.rb#L5

You need to test each method.